signer/local/local.go

```diff
@@ -260,6 +261,47 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 		safeTemplate.SerialNumber = serialNumber
 	}
 
+	if len(profile.AllowedExtensions) > 0 && len(req.Extensions) > 0 {
+		extFilter := map[string]bool{}
+		for _, oid := range profile.AllowedExtensions {
+			extFilter[asn1.ObjectIdentifier(oid).String()] = true
```

Comment:
Maybe this should be set up when the profile is loaded, and an "ExtensionAllowed" method added to avoid having to construct this table on each signature.